assignment.js

```diff
@@ -112,6 +131,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+let playerAChoice = "scissors"
```

Comment:
@kertcodes nice approach to aggregate all a player's conditions into a single clause. Normally, I advise against that, but the code was still easy to follow.